scripts/qa/install_apps.py

```diff
@@ -73,56 +103,78 @@ def domain(self) -> str:
     },
 )
 
-
 class Config(TypedDict):
     """Holds secrets"""
 
     kubeloginClientSecret: str
     adminGroup: str
     dex: DexSecrets
-    externalDns: ExternalDnsConfig
+    dnsProvider: DnsProviderConfig
+    externalLoadbalancers: Optional[ExternalLoadbalancers]
     objectStorage: StorageSecrets
 
     wcSubnets: Subnets
     scSubnets: Subnets
 
 
-def initialize_apps() -> None:
-    """Initialize apps"""
-    _run_ck8s("init", "both")
-
-
-def set_objectstorage_secrets(secrets_path: Path, secrets: StorageSecrets) -> None:
-    """Set objectstorage secrets"""
-    _set_secret_key(secrets_path, '["objectStorage"]', dict(secrets))
-
-
-def update_ips(cluster: str) -> None:
-    """Update IPs"""
-    _run_ck8s("update-ips", cluster, "apply")
-
-
 def configure_apps(
     common_config: AppsConfig, sc_config: AppsConfig, wc_config: AppsConfig, args: Args
 ) -> None:
     """Configure apps"""
 
-    opensearch_cfg = cast(dict, sc_config.get("opensearch"))
+    # Configure platform administrators
+    common_config.set("clusterAdmin", {"users": [], "groups": [args.config["adminGroup"]]})
 
-    # Set domains
+    # Configure application developers
+    wc_config.set(
+        "user",
+        {
+            "namespaces": ["production", "staging"],
+            "adminUsers": ALL_USERS,
+            "adminGroups": [],
+        },
+    )
+
+    # Configure domains
     common_config.set(
         "global",
         {"baseDomain": args.domain, "opsDomain": f"ops.{args.domain}"},
     )
 
+    # Configure external loadbalancer
+    try:
+        cond = args.config["externalLoadbalancers"]["scProxyProtocol"]
+        sc_config.set("ingressNginx.controller.config", {"useProxyProtocol": cond})
+    except KeyError:
+        pass # Use default
+    try:
+        cond = args.config["externalLoadbalancers"]["wcProxyProtocol"]
+        wc_config.set("ingressNginx.controller.config", {"useProxyProtocol": cond})
+    except KeyError:
+        pass # Use default
+
+    # Configure object storage
+    try:
+        region = args.config["objectStorage"]["s3"]["region"]
+        if region != "":
+            common_config.set("objectStorage.s3", {"region": region})
+    except KeyError:
+        pass # Use default
+    try:
+        regionEndpoint = args.config["objectStorage"]["s3"]["regionEndpoint"]
+        if region != "":
+            common_config.set("objectStorage.s3", {"regionEndpoint": regionEndpoint})
+    except KeyError:
+        pass # Use default
+
```

Comment:
If you'd like a more ergonomic `try` with empty `catch` handler:

```suggestion
    # Configure external loadbalancer
    with suppress(KeyError):
        cond = args.config["externalLoadbalancers"]["scProxyProtocol"]
        sc_config.set("ingressNginx.controller.config", {"useProxyProtocol": cond})

    with suppress(KeyError):
        cond = args.config["externalLoadbalancers"]["wcProxyProtocol"]
        wc_config.set("ingressNginx.controller.config", {"useProxyProtocol": cond})

    # Configure object storage
    with suppress(KeyError):
        region = args.config["objectStorage"]["s3"]["region"]
        if region != "":
            common_config.set("objectStorage.s3", {"region": region})

    with suppress(KeyError):
        regionEndpoint = args.config["objectStorage"]["s3"]["regionEndpoint"]
        if regionEndpoint != "":
            common_config.set("objectStorage.s3", {"regionEndpoint": regionEndpoint})
```

it needs an extra `from contextlib import suppress` at the top

Reply:
Nice, will do!